### PR TITLE
test(richtext): use embed dialogs for testing the correct behaviour []

### PR DESCRIPTION
**Pull request by chrishelgert**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -77,6 +77,14 @@ jobs:
               --reporter junit \
               --reporter-options "mochaFile=./cypress/reports/component/test-results.[hash].xml" \
               --browser chrome
+      - vault/get-secrets:
+          template-preset: packages-read
+      - run: |
+          echo "//npm.pkg.github.com/:_authToken=${GITHUB_PACKAGES_READ_TOKEN}" > ~/.npmrc
+          echo "@contentful:registry=https://npm.pkg.github.com" >> ~/.npmrc
+      - run:
```
Comment (chrishelgert) [inline .circleci/config.yml:85]: After this the splitting into wrokers should work correctly

**Review by z0al — APPROVED** [review]
LGTM. Btw, there is a `test` commit type for these kind of changes :)

File: cypress.config.ts [hunk 2]
```diff
@@ -23,27 +23,6 @@ export default defineConfig({
     openMode: 0,
   },
   numTestsKeptInMemory: 1,
-  e2e: {
```
Comment (chrishelgert) [inline cypress.config.ts:26]: We don't have e2e test anymore 😈

File: cypress/component/rich-text/RichTextEditor.Commands.spec.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ import { mountRichTextEditor } from './utils';
 // the sticky toolbar gets in the way of some of the tests, therefore
 // we increase the viewport height to fit the whole page on the screen
 
-describe('Rich Text Editor - Commands', { viewportHeight: 2000 }, () => {
+describe('Rich Text Editor - Commands', { viewportHeight: 2000, viewportWidth: 1000 }, () => {
```
Comment (chrishelgert) [inline cypress/component/rich-text/RichTextEditor.Commands.spec.ts:11]: All RichText texts are using the same width and height

**Pull request merged**